packages/ui-components/src/components/layout/Dialog.vue

```diff
@@ -27,21 +27,35 @@
             leave-to="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
             @after-leave="$emit('fully-closed')"
           >
-            <DialogPanel
-              :class="[
-                'transform rounded-lg bg-foundation text-left shadow-xl transition-all',
-                widthClasses
-              ]"
-            >
-              <div class="relative">
+            <DialogPanel :class="dialogPanelClasses">
+              <div
+                v-if="title"
+                class="flex items-center justify-center shadow p-4 relative z-10 bg-foundation"
+              >
+                <h4 class="text-2xl font-bold">{{ title }}</h4>
+              </div>
+              <button
```

Comment:
You should use `FormButton` wherever possible

Reply:
I'm not sure why this is highlighted as a change. I didn't change this, and its a <button> on main. I can take a look at changing it to FormButton if you want?

Reply:
Aren't these the new buttons you added to the dialog component?

Reply:
Ah right, that button already existed before. And it wasn't a FormButton because it was supposed to be a very simple closer button.